docs/overview.md

```diff
@@ -64,8 +64,8 @@ Installing OpenEBS in your cluster is as simple as a few `kubectl` or `helm` com
 
 ### Local Volumes
 
-- [Local PV hostpath](/docs/next/uglocalpv-hostpath.html)
-- [Local PV device](/docs/next/uglocalpv-device.html)
+- [Local PV hostpath](https://github.com/openebs/dynamic-localpv-provisioner/blob/develop/docs/quickstart.md)
```

Comment:
@niladrih - instead of this, can we actually update the uglocalpv-device.html and uglocalpv-hostpath.html with missing details and also add a note about -- checking out the repo for more details.